api/capi/src/nnstreamer-capi-util.c

```diff
@@ -1187,8 +1187,8 @@ ml_get_nnfw_subplugin_name (ml_nnfw_type_e nnfw)
     [ML_NNFW_TYPE_VIVANTE] = "vivante",
     [ML_NNFW_TYPE_EDGE_TPU] = "edgetpu",
     [ML_NNFW_TYPE_ARMNN] = "armnn",
-    [ML_NNFW_TYPE_SNAP] = "snap",
     [ML_NNFW_TYPE_SNPE] = "snpe",
```

Comment:
How about add `/* Android only */` here for snpe too?

Reply:
we can build snpe subplugin on ubuntu :)

Reply:
Oh that's right!